src/simulator_core/entities/esdl_object.py

```diff
@@ -51,3 +53,30 @@ def get_all_assets_of_type(self, esdl_asset_type: str) -> list[EsdlAssetObject]:
         return [EsdlAssetObject(asset)
                 for asset in self.energy_system_handler.get_all_instances_of_type(
                 StringEsdlAssetMapper().to_esdl(esdl_asset_type))]
+
+    def get_connected_assets(self, id: str, port: Port) -> List[Tuple[str, Port]]:
+        """Method to get the id's of connected assets from the esdl.
+
+        This returns a list of list with the connected asset id and the port to which it is
+        connected to the asset.
+
+        :param str id: id of the asset for which we want to know the connected assets
+        :param Port port: port for which the connected assets need to be returned.
+        :return: List of list which the id of the connected assets and the connected port.
+        """
+        esdl_asset = self.energy_system_handler.get_by_id(id)
+        connected_assets = []
+        type_port = esdl.InPort
```

Comment:
I would move this type_port statement to the if loop.

`
if port == Port.Out:
   type_port = esdl.OutPort
else:
   type_port = esdl.InPort
`

Reply:
made a one line statement out of it.        
type_port = esdl.InPort if port == Port.Out else esdl.OutPort